### ISSUE TITLE
keywords.py AttributeError

### ISSUE THREAD

**Issue by hellrich** [piskvorky/gensim#503]
I tried the example given at http://rare-technologies.com/text-summarization-with-gensim/ and got the following error:

```
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/hellrich/bin/anaconda3/envs/gensimtest/lib/python3.4/site-packages/gensim/summarization/keywords.py", line 198, in keywords
    graph = _build_graph(_get_words_for_graph(tokens))
  File "/home/hellrich/bin/anaconda3/envs/gensimtest/lib/python3.4/site-packages/gensim/summarization/keywords.py", line 35, in _get_words_for_graph
    for word, unit in tokens.iteritems():
AttributeError: 'dict' object has no attribute 'iteritems'
```

**Comment by piskvorky** [comment]
Looks like a python3 error that was recently fixed, in #441 .

Let me know if that worked for you; closing as "already fixed" for now.

**Comment by hellrich** [comment]
Works fine, thanks!
